source/_integrations/iammeter.markdown

```diff
@@ -0,0 +1,81 @@
+---
+title: "iammeter"
+description: "Instructions on how to integrate iammeter sensor within Home Assistant."
+logo: iammeter-logo.png
+ha_category:
+  - Energy
+  - Sensor
+ha_release: 0.104
+ha_iot_class: Local Polling
+---
+
+[https://www.iammeter.com](https://www.iammeter.com) is our energy management platform/cloud. Our Bi-Directional Wi-Fi Energy Meters can measure the real-time reading and report to our cloud. These meters can also be easily integrated into home assistant system. We have both single phase (WEM3080, WEM3162) and three phase (WEM3080T) meters.
```

Comment:
```suggestion
[IAMMETER](https://www.iammeter.com) is our energy management platform/cloud. Our Bi-Directional Wi-Fi Energy Meters can measure the real-time reading and report to our cloud. These meters can also be easily integrated into Home Assistant system. We have both single-phase (WEM3080, WEM3162) and three-phase (WEM3080T) meters.
```

> is our energy management platform/cloud.

> to our cloud

> We have both single-phase

This is the Home Assistant documentation, the way it is written makes it looks like it is ours (Home Assistant). However, it is not. Please rephrase it.